rtl/eclrtl/eclrtl.cpp

```diff
@@ -3652,6 +3652,22 @@ int rtlSingleUtf8ToCodepage(char * out, unsigned inlen, char const * in, char co
     return static_cast<int>(trailbytes); //cast okay as is certainly 0--3
 }
 
+bool rtlCodepageToCodepage(StringBuffer & out, unsigned maxoutlen, unsigned inlen, char const * in, char const * outcodepage, char const * incodepage)
+{
+    UConverter * inconv = queryRTLUnicodeConverter(incodepage)->query();
+    UConverter * outconv = queryRTLUnicodeConverter(outcodepage)->query();
+    UErrorCode err = U_ZERO_ERROR;
```

Comment:
Is it worth copying skipping the conversion when incodepage and out codepage is the same? I know you've mentioned that it's not currently used by the generated code but it could be included for some purposed in the future.

Reply:
I can't imagine this ever being called with the same source and target codepage, so not worth skipping the conversion.